cmd/node/main.go

```diff
@@ -537,23 +542,39 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 		shardId = fmt.Sprintf("%d", shardCoordinator.SelfId())
 	}
 
-	uniqueDBFolder := filepath.Join(
+	pathTemplateForPruningStorer := filepath.Join(
+		workingDir,
+		defaultDBPath,
+		fmt.Sprintf("%s_[E]", defaultEpochString),
```

Comment:
[E], [S], [I] are defined as constants also in the pathManager. Could we define them somewhere in the core package and use them from there everywhere?

Reply:
yes, moved to constants in core

Reply:
ok